### ISSUE TITLE
When dragged to uppering floor in req, crate falls down (still can be dragged after falling) and appears here

### ISSUE THREAD

**Issue by Monikawales** [tgstation/TerraGov-Marine-Corps#947]
## Reproduction:
![image](https://user-images.githubusercontent.com/45321263/53690049-2fdca200-3d74-11e9-9ce0-2f8aebca37b0.png)

**Comment by Monikawales** [comment]
Drag a crate here
![image](https://user-images.githubusercontent.com/45321263/53690054-42ef7200-3d74-11e9-9a47-856d4ea8db09.png)
When platform is raising

**Comment by Alterist** [comment]
Sorta Intended behaviour, anything that falls down the req elevator gets dumped by the chute in the storage room, the real issue is the rails dropping before the elevator is completely up

**Comment by MrStonedOne** [comment]
Please do not leave format template noise in your issue body. Remove any section that does not apply.

**Comment by DominikPanic** [comment]
@Alterist Fix/wontfix?

**Comment by Rohesie** [comment]
This will end up changing with the shuttle rework, as the req platform is an abstract shuttle.
The issue is what Alterist pointed, in any case. The railings should drop only after the shuttle arrives.
@spookydonut , think you can add that behavior?
